Automattic/wp-calypso
# Added MailPoet upsell to email management page

#### Proposed Changes

This PR adds an upsell link for the MailPoet plugin to the email management home screen (`/email/:site_slug`). The link leads to the marketplace page for the MailPoet plugin (`/plugins/mailpoet/:site_slug`) and records a Tracks event when clicked.

It's worth noting that because this is shown on the `/email/:site` page, it will only be visible to users who don't already have an email subscription or to users with multiple domains (because users with a single domain that has email will be automatically redirected from `/email/:site` to `/email/:domain/manage/:site`).

**EDIT:** I've replaced the original screenshot with the latest version.

![mailpoet-upsell](https://user-images.githubusercontent.com/1101677/183841290-256ca15b-5c60-43da-87b1-6e2b00d558c9.png)

#### Testing Instructions



1. Have a site with one custom domain without any existing email subscription, or have a site with multiple custom domains (with or without email subscriptions)
2. Go to Upgrades > Emails
3. Ensure that the MailPoet upsell is displayed as per the screenshot above
4. Ensure that the link leads to the correct destination
5. Ensure that a Tracks event is recorded when clicking the link

#### Pre-merge Checklist






Related to #
